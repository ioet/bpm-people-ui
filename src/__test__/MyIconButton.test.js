import React from 'react';
import BpmIconButton from '../component/presentational/BpmIconButton';

describe('<BpmIconButton />', () => {
  test('Component Renders correctly', () => {
    const appWrapper = shallow(<BpmIconButton />);

    expect(toJson(appWrapper))
      .toMatchSnapshot();
  });
});
