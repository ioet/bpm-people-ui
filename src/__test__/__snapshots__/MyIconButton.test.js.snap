// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BpmIconButton /> Component Renders correctly 1`] = `
<BpmIconButton
  classes={
    Object {
      "hide": "BpmIconButton-hide-1",
      "show": "BpmIconButton-show-2",
    }
  }
  hoverId=""
/>
`;
